lib/pure/typetraits.nim

```diff
@@ -284,3 +295,6 @@ proc hasClosure*(fn: NimNode): bool {.since: (1, 5, 1).} =
   ## arguments and not `untyped` arguments.
   expectKind fn, nnkSym
   result = hasClosureImpl(fn)
+
+from std/private/bitops_utils import toUnsigned, toSigned
+export toUnsigned, toSigned
```

Comment:
This is not a "type trait", it seems wiser to publish "bitops_utils" instead, somehow.

Reply:
D defines those APIs in `std.traits`
C++ defines in `std::type_traits`
and we already have APIs like `distinctBase`, `elementType` in std/typetraits; it's much simpler to have these APIs (type => type) in `std/typetraits` than to create a dedicated bitops_utils module or equivalent (we already have bitops, and toUnsigned/toSigned is a poor fit for bitops) whose scope would be unclear.

Reply:
Can you move them to typetraits directly then?